lang/pt_BR.json

```diff
@@ -312,19 +312,19 @@
     "Someone follows me": "Alguém me segue",
     "Something went wrong.": "Algo deu errado.",
     "Sort order": "Ordem de classificação",
-    "Started": "Started",
-    "Statistics": "Statistics",
+    "Started": "Inicado",
```

Comment:
possible typo: should this be "Iniciado"?

Reply:
this is correct